src/asm/warning.c

```diff
@@ -70,6 +71,7 @@ static char const *warningFlags[NB_WARNINGS_ALL] = {
 	"assert",
 	"builtin-args",
 	"charmap-redef",
+	"comment-in-block-comment",
```

Comment:
This seems fairly long, especially compared to the `nested-comment` I had suggested earlier. The length is further evidenced through it throwing other warning constants out of alignment, which I have been considering to be a soft cap.

(I'm not fond of the length of e.g. `empty-data-directive`, but too late to turn back)